typescript/cli/ci-test.sh

```diff
@@ -163,6 +163,7 @@ MESSAGE1_ID=`cat /tmp/message1 | grep "Message ID" | grep -E -o '0x[0-9a-f]+'`
 echo "Message 1 ID: $MESSAGE1_ID"
 
 WARP_ARTIFACTS_FILE=`find /tmp/warp-route-deployment* -type f -exec ls -t1 {} + | head -1`
```

Comment:
Totally optional but we could probably cut this and just use the WARP_CONFIG_FILE instead. Though it may be more cumbersome to extract a value from here in bash so feel free to ignore.

Reply:
jq to the rescue!